tests/worksheet_test.py

```diff
@@ -629,6 +629,45 @@ def test_get_all_records_duplicate_keys(self):
         with pytest.raises(GSpreadException):
             self.sheet.get_all_records()
 
+    @pytest.mark.vcr(allow_playback_repeats=True)
+    def test_get_all_records_expected_headers(self):
+        self.sheet.resize(4, 4)
+
+        # put in new values
+        rows = [
+            ["A1", "B2", "C3", "D4"],
+            [1, "b2", 1.45, ""],
+            ["", "", "", ""],
+            ["A4", 0.4, "", 4],
+        ]
+        cell_list = self.sheet.range("A1:D4")
+        for cell, value in zip(cell_list, itertools.chain(*rows)):
+            cell.value = value
+        self.sheet.update_cells(cell_list)
```

Comment:
[idea] The tests might be simpler to read if this were moved to a helper method like `update_sheet_cell_values` (or something like that).

Reply:
you're right but this will be done during some refactoring of the test suite. with generic helper functions to generate data etc.

thank you for you feedback.